theme/acquia_lift.wizard.theme.inc

```diff
@@ -45,6 +46,9 @@ function theme_acquia_lift_card($variables) {
   }
   $html .= '</div>';
   $html .= '<div class="el-card__content">';
+  if (!empty($card_flag)) {
```

Comment:
It is certainly a pattern that we have not worked a lot on yet. `.is-active` Would definitely be good for the element being dragged as it has a drop shadow and will show a form of lift. In a pinch the same could be used on the targets until we think of something better.